livro/styles/cream.json

```diff
@@ -0,0 +1,36 @@
+{
+	"version": 2,
+	"title": "Cream",
+	"settings": {
+		"appearanceTools": true,
```

Comment:
Since `appearanceTools` is true in the default styles I think this can be eliminated in both of these variations.